app/javascript/components/SearchPanel.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import FacetControl from './FacetControl';
 import MoreFacetsButton from './MoreFacetsButton';
 import DownloadButton from './DownloadButton';
+import KeywordSearch from './KeywordSearch';
```

Comment:
There seems to be a bug here: 

`KeyWordSearch` is exported from `KeywordSearch.js`, but this imports `KeywordSearch`.  The class `KeywordSearch` is undefined.

I'd update the class name to `KeywordSearch`, as we naturally write this thing as "keyword" and not "key word".

Reply:
Addressed in 8e23d95